spec/steps/income_details_per_member_spec.rb

```diff
@@ -1,17 +1,28 @@
 require "rails_helper"
 
 RSpec.describe IncomeDetailsPerMember do
-  describe "#working_members" do
-    it "returns members who are employed or self employed" do
-      working = create(:member, employment_status: "employed")
-      not_working = create(:member, employment_status: "not_employed")
-      self_employed = create(:member, employment_status: "self_employed")
-      members = [working, not_working, self_employed]
+  describe "Validations" do
+    it "validates employer name on employed members" do
+      working = create(
+        :member,
+        employment_status: "employed",
+        employed_employer_name: nil,
+      )
+      self_employed = create(
+        :member,
+        employment_status: "self_employed",
+        employed_employer_name: nil,
+      )
+      members = [working, self_employed]
       snap_application = create(:snap_application, members: members)
 
       step = IncomeDetailsPerMember.new(members: snap_application.members)
+      step.valid?
 
-      expect(step.working_members).to match_array([working, self_employed])
+      expect(step.members.last.errors[:employed_employer_name]).
+        to eq([])
+      expect(step.members.first.errors[:employed_employer_name]).
+        to eq(["Make sure to answer this question"])
```

Comment:
I think(?) the form objects automatically handle displaying errors, yeah? If we bubble up the errors to the step object itself I believe it'll handle it without us reaching down into the associated `members` objects to check for errors. From the step's perspective that would mean all errors would be accessible via the step object itself.

```ruby
expect(step.errors[:employed_employer_name]).
  to eq(["Make sure to answer this question"])
```

I'll pull this down and play with it a little bit to see if I'm wrapping my head around this completely. I might be missing a bit by just looking at the diff :).

Reply:
in order for the errors to show correctly on the form, it needs to be on a `member`

Reply:
Ah-ha! Ok, cool

Reply:
the reason it is bubbling up to the parent `step` is that we do this check inside the `validate` method